modules/services/window-managers/i3-sway/i3.nix

```diff
@@ -130,6 +137,19 @@ let
           for more advanced setup use 'i3.extraConfig'.
         '';
       };
+
+      variables = mkOption {
```

Comment:
I don't know anything about i3 but this doesn't seem to make any sense. Why would you want to define variables as an option?